frontend/src/features/workspace/WorkspaceContext.tsx

```diff
@@ -0,0 +1,29 @@
+import { createContext, useContext } from 'react'
+
+import { AdminDashboardFormMetaDto } from '~shared/types'
+
+import { FilterOption } from './types'
+
+export interface WorkspaceContextProps {
+  isLoading: boolean
+  totalFormsCount?: number
+  displayedForms: AdminDashboardFormMetaDto[]
+  displayedFormsCount: number
+  isFilterOn: boolean
+  activeFilter: FilterOption | null
```

Comment:
feels like the presence of `activeFilter` will === `isFilterOn`, do we need this prop?

edit: did a cursory check, indeed it is just an alias. if having no filter === null anyways, we don't really need this prop imho and can just use the presence as a check

Reply:
[69ff759](https://github.com/opengovsg/FormSG/pull/5234/commits/69ff759bf55e3b215ef3e4a51189a12e70d4ea4e)